chapters/order.txt

```diff
@@ -20,6 +20,7 @@ Graphics
 	lines
 	generativemesh
 	advanced_graphics
+    textures
```

Comment:
can you fix the indentation here? i believe this should be a tab instead of four spaces but turning everything to spaces would be ok too